validphys2/src/validphys/closuretest/multiclosure_output.py

```diff
@@ -60,6 +60,65 @@ def plot_total_fits_bias_variance(fits_total_bias_variance):
     """
     return plot_dataset_fits_bias_variance(fits_total_bias_variance, "all data")
 
+@table
+def table_datasets_bias_variance_fits(fits_datasets_bias_variance, each_dataset):
+    """
+    Table with ratio bias variance value and associated uncertainty
+    computed with simple gaussian error propagation for each dataset.
+    """
+    records = []
+    # compute expectation and uncertainty on bias variance ratio
+    for ds, (bias, var, ndata) in zip(each_dataset, fits_datasets_bias_variance):
+        mean_bias = np.mean(bias)
+        mean_var = np.mean(var)
+        rbv = mean_bias / mean_var
+
+        # compute uncertainy on rbv
+        delta_rbv = np.sqrt(
+            ((1 / mean_var) * np.std(bias)) ** 2 + (mean_bias / mean_var**2 * np.std(var)) ** 2
+        )
+        sqrt_rbv = np.sqrt(rbv)
+        delta_sqrt_rbv = 0.5 * delta_rbv / np.sqrt(rbv)
```

Comment:
This block is used almost unchanged at least twice, maybe it would be sensible to have a unit function for that